moto/autoscaling/models.py

```diff
@@ -504,19 +504,22 @@ def update_attached_elbs(self, group_name):
         group_instance_ids = set(
             state.instance.id for state in group.instance_states)
 
-        try:
-            elbs = self.elb_backend.describe_load_balancers(
-                names=group.load_balancers)
-        except LoadBalancerNotFoundError:
-            # ELBs can be deleted before their autoscaling group
-            return
-
-        for elb in elbs:
-            elb_instace_ids = set(elb.instance_ids)
-            self.elb_backend.register_instances(
-                elb.name, group_instance_ids - elb_instace_ids)
-            self.elb_backend.deregister_instances(
-                elb.name, elb_instace_ids - group_instance_ids)
+        # skip this if group.load_balancers is empty
+        # otherwise elb_backend.describe_load_balancers returns all available load balancers
+        if group.load_balancers:
```

Comment:
How about just returning here if group.load_balancers is empty? Something like:

```
if not group.load_balancers:
    return
```
That'll save a level of indentation. If you prefer it the current way I'll defer to you and merge it.